README.md

```diff
@@ -70,7 +70,7 @@ To get you started, we used actions to go ahead and made a branch and pull reque
 1. Open a new browser tab, and navigate to this same repository. Then, work on the steps in your second tab while you read the instructions in this tab.
 1. Navigate to the **Code** tab.
 1. From the **main** branch dropdown, click on the **welcome-workflow** branch.
-1. Navigate to the `.github/workflows/` folder, then select **Add file** and click on **Create new file**.
+1. Navigate to the `.github/workflows/` folder, then select **Add file** and click on **+ New file**.
```

Comment:
The correct button we would find is indeed "Create new file", not "+ New file".

In fact, the "Add file" label is hidden and pops up only on mouse hover.

IMHO, a better fix to make it clearer would be:

```
1. Navigate to the `.github/workflows/` folder, then select the "**+**" sign (Add file) on the top right and click on **Create new file**.
```

For quick check:

![image](https://user-images.githubusercontent.com/86871991/202951935-e576e22b-23f7-4236-b8eb-5ff7a19053fe.png)